pkgs/development/libraries/glib-networking/default.nix

```diff
@@ -28,6 +28,11 @@ stdenv.mkDerivation rec {
   ];
   propagatedBuildInputs = [ glib gnutls p11-kit libproxy gsettings-desktop-schemas ];
 
+  mesonFlags = [
+    # Default auto detection doesn't work
+    "-Dgnutls=enabled"
```

Comment:
So, we might want `-Dgnutls=disabled` and  `-Dopenssl=enabled`, that should apparently fix the telepathy-idle cpu issue: https://gitlab.gnome.org/GNOME/glib-networking/issues/69

Reply:
It is still unstable, though.